.gitignore

```diff
@@ -11,3 +11,6 @@ build
 *.pfs0
 *.nacp
 *.nro
+test.*
+*_bin.h
```

Comment:
remove

Reply:
do you want to remove the switch folder too? i mainly added these because if someone builds on a mac/pc these files are created... and this .gitignore prevents those files from being added. i know last time you had me delete the files

Reply:
You can leave the other lines.

Reply:
a test.dsymc is created which contains a lot of the application data is created on the mac... and a switch folder is also created.

Reply:
ok, so remove test.* and *_bin.h ?

Reply:
Just remove the bin.h line.